### ISSUE TITLE
Remote cwctl project sync fails because docker isn't running locally

### ISSUE THREAD

**Issue by tetchel** [eclipse/codewind#2807]
```
[April 30, 10:03:48.937] [1419.278] Received event [MODIFY] /Users/tim/vscode-codewind-workspace/nodej/server/routers/codewind.js
[April 30, 10:03:49.945] [1420.286] Batch change summary for 5c94e8d0-8ae8-11ea-89dc-fd0da1fec724 @ 1588255428937: [ >codewind.js ]
[April 30, 10:03:49.946] [1420.287] Calling cwctl project sync with: { [--insecure] [project] [sync] [-p] [/Users/tim/vscode-codewind-workspace/nodej] [-i] [5c94e8d0-8ae8-11ea-89dc-fd0da1fec724] [-t] [1588254117785] }
[April 30, 10:03:50.056] [1420.397] ! ERROR !  Error running 'project sync' installer command time="2020-04-30T10:03:50-04:00" level=error msg="{\"error\":\"config_connection_notfound\",\"error_description\":\"{\\\"error\\\":\\\"CONTAINER_LIST_ERROR\\\",\\\"error_description\\\":\\\"Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?\\\"}\"}"

[April 30, 10:03:50.056] [1420.397] ! ERROR !  Stdout:No output
[April 30, 10:03:50.057] [1420.398] ! ERROR !  Stderr:time="2020-04-30T10:03:50-04:00" level=error msg="{\"error\":\"config_connection_notfound\",\"error_description\":\"{\\\"error\\\":\\\"CONTAINER_LIST_ERROR\\\",\\\"error_description\\\":\\\"Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?\\\"}\"}"

[April 30, 10:03:50.057] [1420.398] !!! SEVERE !!!  Unexpected exception from CLI:
[object Object]
```

Verifying https://github.com/eclipse/codewind/issues/2785 I see the above issue. I have minikube running but not docker, which causes project sync to fail on my remote (minikube) cluster.

**Comment by tetchel** [comment]
See #2845, this causes any `cwctl project` command to fail

**Comment by stalleyj** [comment]
The fix is now in master and 0.12.0

**Comment by tetchel** [comment]
:+1:
